Assets/Scripts/UI/MenuController.cs

```diff
@@ -1,15 +1,135 @@
+using Mirror;
+using TMPro;
 using UnityEngine;
 using UnityEngine.SceneManagement;
+using UnityEngine.UI;
 
-public class MenuController : MonoBehaviour
+namespace UI
 {
-    public void PlayGame()
+    public class MenuController : MonoBehaviour
```

Comment:
TBH just copied all this code from https://mirror-networking.gitbook.io/docs/community-guides/unity-canvas-hud-guide